train_sentiment_model.py

```diff
@@ -0,0 +1,18 @@
+from sentiment_pretraining_defaults import *
```

Comment:
There seems to be some inconsistencies in in which metadata files are kept in the repo. votes-2008-10, for instance, is included below for some reason

Reply:
That's strange, since votes/ is being ignored. This is what we are currently ignoring. Please lmk if anything else should be included

stanford-corenlp-4.0.0
votes
*.DS_Store
test.csv
timedict
total_count
train.csv
t_sentiments
v_sentiments
c_sentiments
subreddit

cache_dir
outputs
runs
/__pycache__/
*.pyc
sentiment_pretraining/*.txt
sentiment_pretraining/*.csv
timedict
author
auto_labels
total_count
counts
Human_Ratings
__pycache__
avg_textblob_vader
*.csv
subreddit

Reply:
Also bert_prep should be included. It might have been in the later "model_improvements"

Reply:
sentiments/ folder should be included

Reply:
> The output folders should be excluded too, but then their names can be varied (e.g. base_persuasion_pre_False_lda_False...). An easier alternative would be to set "output_path" in defaults.py's uploaded version to a folder outside of the "repoed" folder. Does that makes sense?

Can we just ignore all files with the 'model_path' prefix, since that's how output_path is created?

Reply:
Although model_path is machine specific.